controllers/passenger.py

```diff
@@ -0,0 +1,42 @@
+from fastapi import APIRouter, Depends
+
+from component_factory import get_passenger_service
+
+from model.requests.passenger import PassengerDriveOrderRequest, PassengerGetDrive
+from model.drive_order import DriveOrder
+from model.responses.passenger import DriveOrderResponse, GetDriveResponse
+from service.passenger_service import PassengerService
+
+router = APIRouter()
+
+
+def _to_drive_order_response(order: DriveOrder):
+    return DriveOrderResponse(order_id=order.id)
+
+
+def _to_get_drive_response(drive_id: int):
+    return GetDriveResponse(drive_id=drive_id)
+
+
+def _address_to_location(address: str):
```

Comment:
use existing geocoding service

Reply:
so Ofir and I have agreed on the following contract: FE sends BE each location by its lon and lat and not by address, so BE only knows "points" and not real addresses. Meaning this function is redundant (i'm updating the rest of the code according to this new agreement)